src/remage-doc-dump.cc

```diff
@@ -63,6 +64,7 @@ void init_extra() {
   new RMGIsotopeFilterScheme();
   new RMGTrackOutputScheme();
   new RMGParticleFilterScheme();
+  new RMGGeomBenchOutputScheme();
```

Comment:
this doesn't even define commands, so you can remove this here and also the `DefineCommands` in `RMGGeomBenchOutputScheme`...?